packages/css/src/components/Toggle.css

```diff
@@ -0,0 +1,101 @@
+.Toggle {
+  position: relative;
+  width: 54px;
+  height: 20px;
+  background-color: var(--color-white);
+  border: 1px solid var(--color-chatelle);
+  border-radius: 10px;
+  cursor: pointer;
+  transition: all 0.3s;
+}
+
+.Toggle:hover {
+  border: 1px solid var(--color-granite);
+}
+
+.Toggle .Toggle--bullet {
+  position: absolute;
+  height: 14px;
+  width: 14px;
+  background-color: var(--color-granite);
+  border-radius: 8px;
+  top: 2px;
+  left: 2px;
+  transition: all 0.3s;
+}
+
+.Toggle--bullet-label {
+  text-transform: uppercase;
+  text-align: center;
+  font-size: var(--font-size--xs);
+  line-height: var(--line-height--s);
+  position: absolute;
+  top: 2px;
+  bottom: 2px;
+}
+
+.Toggle .Toggle--bullet-label.is-off {
+  left: 18px;
+  right: 4px;
+  color: var(--color-granite);
+  visibility: visible;
+  transition: visibility 0.3s linear;
+  transition-delay: 0.1s;
+}
+
+.Toggle.is-active .Toggle--bullet-label.is-off {
+  visibility: hidden;
+  transition: visibility 0.1s linear;
+}
+
+.Toggle .Toggle--bullet-label.is-on {
+  right: 18px;
```

Comment:
This could go into a variable as it's also used into `.Toggle--bullet-label.is-off`

Reply:
What would the variable be?

Reply:
🆙 , it will help scalability

Reply:
👍